source/ed/connectors/fusio_parser.h

```diff
@@ -88,7 +88,9 @@ struct RouteFusioHandler : public GenericHandler {
     int ignored;
     void init(Data&);
     void handle_line(Data& data, const csv_row& line, bool is_first_line);
-    const std::vector<std::string> required_headers() const { return {"route_id", "route_name", "line_id"}; }
+    const std::vector<std::string> required_headers() const {
+        return {"route_id", "route_name", "line_id", "direction_type"};
```

Comment:
I don't think the direction_type is required.

what do you think about this @prhod ?